packages/angular_devkit/schematics/tools/description.ts

```diff
@@ -25,6 +25,7 @@ export interface FileSystemSchematicJsonDescription {
   readonly factory: string;
   readonly description: string;
   readonly schema?: string;
+  readonly extends?: string;
```

Comment:
This is not specific to file system description, and it's already included in the description type anyway. Remove it from here.